chaosreliably/__init__.py

```diff
@@ -65,56 +50,77 @@ def discover(discover_system: bool = True) -> Discovery:
 ###############################################################################
 # Private functions
 ###############################################################################
-def get_auth_info(configuration: Configuration = None,
-                  secrets: Secrets = None) -> Tuple[str, str]:
+def get_auth_info(
+    configuration: Configuration = None, secrets: Secrets = None
+) -> Dict[str, str]:
     reliably_config_path = None
     reliably_host = None
     reliably_token = None
+    reliably_org = None
 
     configuration = configuration or {}
-    reliably_config_path = os.path.expanduser(configuration.get(
-        "reliably_config_path", RELIABLY_CONFIG_PATH))
+    reliably_config_path = os.path.expanduser(
+        configuration.get("reliably_config_path", RELIABLY_CONFIG_PATH)
+    )
     if reliably_config_path and not os.path.isfile(reliably_config_path):
         reliably_config_path = None
 
     secrets = secrets or {}
     reliably_token = secrets.get("token")
     reliably_host = secrets.get("host")
+    reliably_org = secrets.get("org")
 
     if not reliably_token and reliably_config_path:
-        logger.debug("Loading Reliably config from: {}".format(
-            reliably_config_path))
+        logger.debug("Loading Reliably config from: {}".format(reliably_config_path))
         with open(reliably_config_path) as f:
             try:
                 config = yaml.safe_load(f)
             except yaml.YAMLError as ye:
                 raise ActivityFailed(
                     "Failed parsing Reliably configuration at "
-                    "'{}': {}".format(reliably_config_path, str(ye)))
+                    "'{}': {}".format(reliably_config_path, str(ye))
+                )
         reliably_host = reliably_host or RELIABLY_HOST
         logger.debug("Connecting to Reliably: {}".format(reliably_host))
         auth_hosts = config.get("auths", {})
         for auth_host, values in auth_hosts.items():
             if auth_host == reliably_host:
                 reliably_token = values.get("token")
                 break
-
-    if not reliably_config_path and not reliably_token and not reliably_host:
+        current_org = config.get("currentOrg")
+        if current_org:
+            reliably_org = current_org.get("name")
+
+    if (
+        not reliably_config_path
+        and not reliably_token
+        and not reliably_host
+        and not reliably_org
+    ):
```

Comment:
see this is wher I dislike black... sigh :D I have to let go

Reply:
Better one opinion to rule them all, than many opinions across files 🤣